Exercise_5.java

```diff
@@ -1,20 +1,52 @@
-class IterativeQuickSort { 
+class IterativeQuickSort {

```

Comment:
Please describe your approach briefly.

Reply:
Sure Will do it from upcoming Pr's. Thank you for your review.